torch/_inductor/freezing.py

```diff
@@ -185,12 +186,15 @@ def freeze(
         aot_autograd_gm, params_flat, fw_metadata
     )
 
-    # TODO - further restrict cse ? right now needed to dedup aliasing ops
-    cse_graph = fx_graph_cse(aot_autograd_gm.graph)
-    aot_autograd_gm.graph = cse_graph
-    aot_autograd_gm.recompile()
-
     aot_example_inputs = [example_inputs[ind] for ind in preserved_arg_indices]
+    # For symbolic shape path, there may have TypeError: unhashable type: 'SymInt'.
```

Comment:
Is it possible to support hashing for SymInt (and also other Sym types) too or revise the `fx_graph_cse` in a way to avoid `TypeError`? Then, we don't have to disable CSE here.

Reply:
+1 to this

Reply:
Now, using  Sym types' node as hashing value